deploy/non-olm/v1.2.0/operator.yml

```diff
@@ -143,6 +143,13 @@ rules:
   - '*'
   verbs:
   - '*'
+- apiGroups:
```

Comment:
First don't edit the old version(s) of the operator.yml's. Update `deploy/non-olm/latest/operator.yml`instead of `deploy/non-olm/v1.2.0/operator.yml` .

This also won't work. It is the Role (namespaced permission) not the ClusterRole (global permission). If you rebase the ClusterRole is in the latest/operator.yml added and used now instead of cluster-admin so you can move it there.